Source/codec.cpp

```diff
@@ -166,15 +166,15 @@ void __fastcall codec_encode(void *pbSrcDst, int size, int size_64, char *pszPas
 				++v7;
 			}
 			while ( v7 < 64 );
-			memset(dst, 0, 0x14u);
+			memset(dst, 0, sizeof(dst));
 			memcpy(v4, v9, 0x40u);
 			v4 += 64;
 			v12 -= v6;
 		}
 		while ( v12 );
 		v5 = v6;
 	}
-	memset(v9, 0, 0x80u);
+	memset(v9, 0, sizeof(v9);

```

Comment:
@nomdenom FYI missing a `)` here which broke CI.

Reply:
Argh, fix probably got lost in a rebase somewhere, have too many branches...